src/winter_onboarding_2021/fleet_management_service/models/cab.clj

```diff
@@ -19,4 +20,12 @@
   (:count (first (cab-db/get-count))))
 
 (defn update! [id cab]
-  (cab-db/update! (java.util.UUID/fromString id) cab))
+  (cab-db/update! id cab))
+
+(defn get-by-licence-plate [licence-no]
```

Comment:
Do we need this if we already have `get-by-id-or-licence-plate`

Reply:
`get-by-id-or-licence-plate` is using `get-by-id` & `get-by-licence-plate` internally

Reply:
I was thinking of some generic function, which will accept a hashmap